.github/workflows/create-fe-release.yml

```diff
@@ -0,0 +1,111 @@
+name: Create Frontend Release Action
+
+on:
+  workflow_dispatch:
+    inputs:
+      repository:
+        description: 'GIT Repository project to run the workflow'
+        required: true
+        type: string
+      tag:
```

Comment:
We pass the tag version as parameter and this can be source of human error. I would let the tag be calculated automatically